src/colmap/estimators/covariance.cc

```diff
@@ -95,7 +95,8 @@ bool ComputeSchurComplement(
     point_covs.reserve(points.size());
   }
 
-  VLOG(2) << "Schur elimination of point parameters";
+  VLOG(2) << StringPrintf("Schur elimination of point parameters (n = %d)",
```

Comment:
Nit: I suggest directly just using string streams to compose the strings instead of the extra hop through StringPrintf